packages/eddsa-ticket-pcd/src/EdDSATicketPCD.ts

```diff
@@ -14,8 +14,14 @@ import { v4 as uuid } from "uuid";
 import { EdDSATicketCardBody } from "./CardBody";
 import { getEdDSATicketData, ticketDataToBigInts } from "./utils";
 
+/**
+ * The globally unique type name of the {@link EdDSATicketPCD}.
+ */
 export const EdDSAPCDTypeName = "eddsa-ticket-pcd";
 
+/**
+ * Assigns each currently supported category a unique value.
+ */
```

Comment:
may be worth asking @robknight to help describe what a category actually is.

Reply:
@robknight Do you have any suggestions here?